src/AuditReader.php

```diff
@@ -1013,6 +1013,7 @@ private function createEntity($className, array $columnMap, array $data, $revisi
                 } else {
                     if ($this->loadNativeCollections) {
                         $collection = new PersistentCollection($this->em, $targetClass, new ArrayCollection());
+                        $collection->setOwner($entity, $assoc);
```

Comment:
I had to add this because I was getting this kind of error:

```
Caused by
AssertionError: assert($this->association !== null) in EntityAuditBundle/vendor/doctrine/orm/lib/Doctrine/ORM/PersistentCollection.php:140
Stack trace:
#0 EntityAuditBundle/vendor/doctrine/orm/lib/Doctrine/ORM/PersistentCollection.php(140): assert(false, 'assert($this->a...')
#1 EntityAuditBundle/vendor/doctrine/orm/lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php(508): Doctrine\ORM\PersistentCollection->hydrateAdd(Object(Sonata\EntityAuditBundle\Tests\Fixtures\Relation\OwnedEntity4))
#2 EntityAuditBundle/vendor/doctrine/orm/lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php(143): Doctrine\ORM\Internal\Hydration\ObjectHydrator->hydrateRowData(Array, Array)
#3 EntityAuditBundle/vendor/doctrine/orm/lib/Doctrine/ORM/Internal/Hydration/AbstractHydrator.php(166): Doctrine\ORM\Internal\Hydration\ObjectHydrator->hydrateAllData()
#4 EntityAuditBundle/vendor/doctrine/orm/lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php(967): Doctrine\ORM\Internal\Hydration\AbstractHydrator->hydrateAll(Object(Doctrine\DBAL\Result), Object(Doctrine\ORM\Query\ResultSetMapping), Array)
#5 EntityAuditBundle/vendor/doctrine/orm/lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php(978): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->loadCollectionFromStatement(Object(Doctrine\ORM\Mapping\ManyToManyInverseSideMapping), Object(Doctrine\DBAL\Result), Object(Doctrine\ORM\PersistentCollection))
#6 EntityAuditBundle/src/AuditReader.php(1184): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->loadManyToManyCollection(Object(Doctrine\ORM\Mapping\ManyToManyInverseSideMapping), Object(Sonata\EntityAuditBundle\Tests\Fixtures\Relation\OwnerEntity), Object(Doctrine\ORM\PersistentCollection))
#7 EntityAuditBundle/src/AuditReader.php(328): SimpleThings\EntityAudit\AuditReader->createEntity('Sonata\\EntityAu...', Array, Array, 1)
#8 EntityAuditBundle/tests/RelationTest.php(349): SimpleThings\EntityAudit\AuditReader->find('Sonata\\EntityAu...', Array, 1)
#9 EntityAuditBundle/vendor/phpunit/phpunit/src/Framework/TestCase.php(1612): Sonata\EntityAuditBundle\Tests\RelationTest->testManyToMany()
```
cc @greg0ire just in case you want to take a look

Reply:
@franmomu thanks for letting me know, I opened https://github.com/doctrine/orm/pull/11071 to get that reverted.